src/modules/ens/ens.service.ts

```diff
@@ -181,15 +179,15 @@ export class EnsService {
     owner,
   }: {
     hash: string;
-    name: string;
+    name?: string;
```

Comment:
```suggestion
```
Can we remove `name` as a param as we can read it in this event handler if we need it?

Reply:
sure!